### ISSUE TITLE
Add back hover states on example items

### ISSUE THREAD

**Issue by ejsandoval** [processing/processing-website#207]
The example items on the Examples index are missing hover state interactions. Add either block background or change of opacity on gover.

**Comment by praveen-rikhari** [comment]
@ejsandoval Is this issue still on? Can i work on this?

**Comment by ejsandoval** [comment]
@praveen-rikhari Yes! It's still ongoing. When navigating the Examples index, we need some hover state feedback for the user. I added a video for reference on where this should have interaction feedback.

https://github.com/processing/processing-website/assets/8853646/92839890-aeb2-4e39-8be3-fa0ae3ae208c

Thank you for addressing this issue!

**Comment by ashwinnmit** [comment]
Hi, I've submitted a PR #554, that addresses the hover animation for the images as discussed in this issue. Let me know if there's any feedback or changes needed. Thanks
